README.md

```diff
@@ -22,12 +22,45 @@ The script serves as installer by default, but can also be used with flags `--up
 
 The agent can run as a standalone binary, as long as configuration files are set up correctly.
 
+### Using a Docker reverse proxy
+To avoid adding the metrikad user to the docker group, you can use a reverse proxy. The reverse proxy filters traffic before it reaches the non-networked Docker socket, preventing the agent process from speaking directly to it. The reverse proxy still needs a user that belongs to the docker group to run it so it can proxy requests to the UNIX socket.
```

Comment:
```suggestion
To avoid adding the `metrikad` user to the `docker` group, you can use a reverse proxy. The reverse proxy filters traffic before it reaches the non-networked Docker socket, limiting the docker API calls the agent can make. Note that the reverse proxy still needs to be executed by a user that belongs to the docker group, so it can proxy requests to the UNIX socket.
```